.github/workflows/self-test.yml

```diff
@@ -2,36 +2,11 @@ name: Self-test the reusable workflow
 on:
   workflow_dispatch:
     inputs:
-      with_coverage:
-        type: boolean
-        required: false
-        default: true
-        description: "Set to 'true' to collect and upload code coverage data. Defaults to 'false'."
-      with_tsan:
-        type: boolean
-        required: false
-        default: true
-        description: "Set to 'true' to run tests with Thread Sanitizer. Defaults to 'true'."
-      warnings_as_errors:
-        type: boolean
-        required: false
-        default: false
-        description: "Set to 'true' to treat warnings as errors. Defaults to 'false'."
-      with_api_check:
-        type: boolean
-        required: false
-        default: true
-        description: "Set to 'true' to run the SwiftPM public API breakage check. Defaults to 'true'."
       with_gh_codeql:
         type: boolean
         required: false
         default: true
         description: "Set to 'true' to run CodeQL checks. Defaults to 'true'."
-      with_linting: 
-        type: boolean
-        required: false
-        default: false
-        description: "Set to 'true' to run swift-format's lint process. Defaults to 'false'."
```

Comment:
Why remove these? These are inputs to the `workflow_dispatch` event, which means "when manually run from the GitHub UI". Just change the defaults.

Reply:
emm right i guess